server/src/repositories/config.repository.spec.ts

```diff
@@ -178,4 +181,41 @@ describe('getEnv', () => {
       expect(getEnv).toThrowError('Invalid worker(s) found: api,microservices,randomservice');
     });
   });
+
+  describe('network', () => {
+    it('should return default network options', () => {
+      const { network } = getEnv();
+      expect(network).toEqual({
+        trustedProxies: [],
+      });
+    });
+
+    it('should parse trusted proxies', () => {
+      process.env.IMMICH_TRUSTED_PROXIES = '10.1.0.0,10.2.0.0';
```

Comment:
Mind making one of those a larger subnet (e.g. /24) for validation or whatever later? :)